src/routes/guides/+page.svelte

```diff
@@ -0,0 +1,203 @@
+<script lang="ts">
+  import SectionTitle from "$lib/components/SectionTitle.svelte"
+  import QRCodeIcon from "$lib/images/card-qr-code-icon.svg?dataurl"
+  import DataSchemasIcon from "$lib/images/card-schemas-icon.svg?dataurl"
+  import ArrowIcon from "$lib/images/link-arrow-icon.svg"
+  import Grid from "$lib/components/Grid.svelte"
+  import Breadcrumbs from "$lib/components/Breadcrumbs.svelte"
+  import Title from "$lib/components/Title.svelte"
+  import TableOfContents from "$lib/components/TableOfContents.svelte"
+  import MailIcon from "$lib/images/mail-icon.svg"
+  import SlackIcon from "$lib/images/slack-icon.svg"
+  import Button from "$lib/components/Button.svelte"
+  import { GUIDES } from "./urls"
+
+  const SLACK_URL = "https://slack.ioxio.com"
+  const EMAIL_URL = "mailto:support@ioxio.com"
+
+  const cards = Object.values(GUIDES)
+
+  const secondaryCards = [
+    {
+      title: "Defining Data Products",
+      href: "https://www.youtube.com/watch?v=yPzN04ICsbw",
+      alt: "QR code",
+      target: "_blank",
+      cardImage: QRCodeIcon,
+    },
+    {
+      title: "Productizing Data",
+      href: "https://www.youtube.com/watch?v=f-f6P_-8zoQ",
+      alt: "QR code",
+      target: "_blank",
+      cardImage: QRCodeIcon,
+    },
+  ]
+  /** @type {import('./$types').PageData} */
+  export let data
+</script>
+
+<div class="wrapper">
+  <TableOfContents>
+    <Title title={`IOXIO Dataspace ®️ guides`} />
```

Comment:
I don't think `IOXIO Dataspace` is a registered trademark, but `IOXIO` should be. And there shouldn't be any space before the `®` symbol. Thus I think this should be spelled as `IOXIO@ Dataspace guides`. Could be good to verify this still, not just trust my memory. We should somewhere have some guideline on this; not sure where though.

Reply:
from what I've found it should be `IOXIO Dataspace™`